<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GuestScreen</class>
 <widget class="QWidget" name="GuestScreen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>768</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Century Gothic</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>GuestScreen</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#backGroundImage{
background-image: url(:/images/images/frmBackGround.png);
}

#GuestScreen {
  border: 0 px
}
</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_5">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="backgroundImage" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">/*QWidget#backgroundImage{
background-image: url(:/images/images/frmBackGround.png);
}*/

background-color: rgb(37, 37, 37);</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>173</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="topLeftLabel">
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap resource="resources.qrc">:/images/images/logo_cribtrak.svg</pixmap>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QWidget" name="widget" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <item>
              <widget class="QLabel" name="lblDate">
               <property name="font">
                <font>
                 <family>Century Gothic</family>
                 <pointsize>12</pointsize>
                 <weight>75</weight>
                 <italic>false</italic>
                 <bold>true</bold>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true">color: rgb(172, 172, 172);
background: rgba(255, 255, 255, 0);</string>
               </property>
               <property name="text">
                <string>Date:  16/12/2016</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="lblTime">
               <property name="font">
                <font>
                 <family>Century Gothic</family>
                 <pointsize>12</pointsize>
                 <weight>75</weight>
                 <bold>true</bold>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true">color: rgb(172, 172, 172);
background: rgba(255, 255, 255, 0);</string>
               </property>
               <property name="text">
                <string>Time:  5:57:10:00</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <property name="sizeConstraint">
         <enum>QLayout::SetMinimumSize</enum>
        </property>
        <item>
         <widget class="QLabel" name="instructionLabel">
          <property name="font">
           <font>
            <family>Century Gothic</family>
            <pointsize>24</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">background: rgba(255, 255, 255, 0);
color: rgb(0, 174, 239);</string>
          </property>
          <property name="text">
           <string>Present ID to Access</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="0,0">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QLabel" name="CustomerLogo">
            <property name="autoFillBackground">
             <bool>false</bool>
            </property>
            <property name="styleSheet">
             <string notr="true">background: rgba(255, 255, 255, 0);</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="textFormat">
             <enum>Qt::PlainText</enum>
            </property>
            <property name="pixmap">
             <pixmap>images/default_logo.png</pixmap>
            </property>
            <property name="scaledContents">
             <bool>false</bool>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="kabDescription">
            <property name="font">
             <font>
              <family>Century Gothic</family>
              <pointsize>20</pointsize>
              <weight>75</weight>
              <italic>false</italic>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">color: rgb(0, 174, 239);
background: rgba(255, 255, 255, 0);</string>
            </property>
            <property name="text">
             <string>#DisplayDescriptionHere#</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="kabSerial">
            <property name="font">
             <font>
              <family>Century Gothic</family>
              <pointsize>20</pointsize>
              <weight>75</weight>
              <italic>false</italic>
              <bold>true</bold>
             </font>
            </property>
            <property name="whatsThis">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-weight:600; color:#00aeef;&quot;&gt;Serial Number&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
            <property name="styleSheet">
             <string notr="true">color: rgb(0, 174, 239);
background: rgba(255, 255, 255, 0);</string>
            </property>
            <property name="text">
             <string>#DisplaySerialNumberHere#</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer_3">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QWidget" name="widget_2" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>633</width>
            <height>300</height>
           </size>
          </property>
          <widget class="QLabel" name="pnlBand">
           <property name="geometry">
            <rect>
             <x>90</x>
             <y>0</y>
             <width>500</width>
             <height>270</height>
            </rect>
           </property>
           <property name="maximumSize">
            <size>
             <width>633</width>
             <height>273</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Century Gothic</family>
             <pointsize>40</pointsize>
             <weight>75</weight>
             <italic>false</italic>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">background: rgba(0, 246, 0);
color: rgba(0, 0, 0);</string>
           </property>
           <property name="text">
            <string>ALL TOOLS IN</string>
           </property>
           <property name="textFormat">
            <enum>Qt::AutoText</enum>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
          <widget class="QPushButton" name="toolsStatusButton">
           <property name="geometry">
            <rect>
             <x>90</x>
             <y>0</y>
             <width>500</width>
             <height>270</height>
            </rect>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="font">
            <font>
             <family>Century Gothic</family>
             <pointsize>40</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>TOOLS OUT</string>
           </property>
          </widget>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QPushButton" name="switchButton">
          <property name="icon">
           <iconset resource="resources.qrc">
            <normaloff>:/images/images/null.png</normaloff>
            <normalon>:/images/images/null.png</normalon>:/images/images/null.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>128</width>
            <height>128</height>
           </size>
          </property>
          <property name="shortcut">
           <string>Ctrl+B</string>
          </property>
          <property name="flat">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="toolFinderButton">
          <property name="autoFillBackground">
           <bool>false</bool>
          </property>
          <property name="styleSheet">
           <string notr="true">height: 120px;
background-color: rgba(255, 255, 255,0);</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="resources.qrc">
            <normaloff>:/images/images/tool_finder.svg</normaloff>:/images/images/tool_finder.svg</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>200</width>
            <height>120</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,1,0">
        <item>
         <widget class="QPushButton" name="rtsButton">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(230, 0, 0);
color: rgb(255, 255, 255);
font: 4475 18pt &quot;MS Shell Dlg 2&quot;;
padding: 3px;
border-radius: 10px;</string>
          </property>
          <property name="text">
           <string>RTS</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0">
          <item>
           <widget class="QLabel" name="vesionNumber">
            <property name="font">
             <font>
              <family>Century Gothic</family>
              <pointsize>8</pointsize>
              <weight>50</weight>
              <italic>false</italic>
              <bold>false</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">font: 8pt &quot;Century Gothic&quot;;color: rgb(255, 255, 255);
background: rgba(255, 255, 255, 0);</string>
            </property>
            <property name="text">
             <string>#DisplayVersionNumberHere#</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="FirmwareVersion">
            <property name="styleSheet">
             <string notr="true">font: 9pt &quot;Century Gothic&quot;;
color: rgb(255, 255, 255);
background: rgba(255, 255, 255, 0);</string>
            </property>
            <property name="text">
             <string>#DisplayFirmwareVersionHere#</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLabel" name="label">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap resource="resources.qrc">:/images/images/henhmantrak_logo_small.png</pixmap>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
     <zorder>verticalSpacer</zorder>
     <zorder>verticalSpacer_2</zorder>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
